### DISCUSSION TITLE
Can't filter the Subject field when searching

### DISCUSSION THREAD

**Discussion by wsyxbcl** [photoprism/photoprism#4108] [Q&A] [answered]
I've been working with images that contain metadata in their 'Subject' field. Despite uploading them and confirming that the 'Subject' field is correctly recognized, I can't filter them out using the `subject:"<value>"` method mentioned in the user guide. 

Is this the excepted behavior that photoprism only support filtering recognized faces other than user provided Subject in metadata, or there's something I missed when filtering images based on the 'Subject' field?

**Comment by graciousgrey** [comment, marked as answer]
The subject search filter is an alias for person and therefore only shows photos to which this person is assigned. It is not linked to the subject field in the photo details, as this field is not used exclusively for people.

https://docs.photoprism.app/user-guide/search/filters/#filter-reference

There is currently no special search filter for the subject field in the photo details. However, if you search without a search filter, the photos that contain the search term in the subject of the photo details will be displayed.

**Reply by wsyxbcl** [reply]
It makes sense, thanks for the reply!
